src/arch/InputHandler/InputHandler_Linux_Joystick.h

```diff
@@ -23,11 +21,15 @@ class InputHandler_Linux_Joystick: public InputHandler
 	static int InputThread_Start( void *p );
 	void InputThread();
 
-	int fds[NUM_JOYSTICKS];
-	int m_iLastFd;
-	RString m_sDescription[NUM_JOYSTICKS];
+	struct FileDescriptor {
+		int file = -1;
```

Comment:
I'd name this field `fd` to be more specific that it's a Linux file handle

Reply:
Fair, updated